src/plugins/faultManagement/pluginSpec.js

```diff
@@ -24,10 +24,22 @@ import {
     createOpenMct,
     resetApplicationState
 } from '../../utils/testing';
-import { FAULT_MANAGEMENT_TYPE } from './constants';
+import {
+    FAULT_MANAGEMENT_TYPE,
+    FAULT_MANAGEMENT_VIEW,
+    FAULT_MANAGEMENT_NAMESPACE
+} from './constants';
 
 describe("The Fault Management Plugin", () => {
     let openmct;
+    const faultDomainObject = {
+        name: 'it is not your fault',
```

Comment:
![sensiblechuckle](https://user-images.githubusercontent.com/12446869/181801806-e6d06bec-262c-4c58-8c07-7cc741a157e8.gif)

Reply:
We have fun here.